sdk/storage/storage-queue/src/Pipeline.ts

```diff
@@ -208,7 +212,12 @@ export function newPipeline(
       : credential
   );
 
-  return new Pipeline(factories, {
+  // when options.httpClient is not specified, passing in a DefaultHttpClient instance to
+  // avoid each client creating its own http client.
+  const newOptions = {
+    ...pipelineOptions,
     httpClient: pipelineOptions.httpClient
-  });
+  };
+
+  return new Pipeline(factories, newOptions);
```

Comment:
Sorry I don't understand the purpose of this change. You haven't changed the `PipelineOptions` right? If you are doing this for future proofing, is it the same as
```
return new Pipeline(factories, pipelineOptions);
```

Reply:
You are right. I wanted to undo my previous change but only undid part of it.